prefixfree.js

```diff
@@ -447,10 +449,24 @@ for (var keyword in keywords) {
 
 var 
 selectors = {
-	':read-only': null,
-	':read-write': null,
-	':any-link': null,
-	'::selection': null
+	':any-link': ':any-link',
+	'::backdrop': '::backdrop',
+
+	':fullscreen': ':fullscreen',
+	':full-screen': ':fullscreen',
+
+	//sigh
+	':placeholder': '::placeholder',
+	'::placeholder': '::placeholder',
+	':input-placeholder': '::placeholder',
+	'::input-placeholder': '::placeholder',
+
+	':read-only': ':read-only',
+	':read-write': ':read-write',
+
+	'::selection': '::selection'
+
+
```

Comment:
I like the idea of a map here, but there should be a default that means "same as key". I don't like this duplication. How about `null` being that value?